02 - Web UI Template/CognitiveSearch.UI/Controllers/HomeController.cs

```diff
@@ -122,5 +123,20 @@ private string GetContainerSasUri()
             sasContainerToken = container.GetSharedAccessSignature(adHocPolicy, null);
             return sasContainerToken;
         }
+
+        [HttpPost]
+        public JObject GetGraphData(string query)
+        {
+            if (query == null)
+            {
+                query = "*";
+            }
+            FacetGraphGenerator graphGenerator = new FacetGraphGenerator(_docSearch);
+            var graphJson = graphGenerator.GetFacetGraphNodes(query, "keyPhrases");
+
+            //JsonResult json = new JsonResult(graphJson.ToString());
```

Comment:
Did you mean to have this line commented out as part of the submission?

Reply:
Comment Removed!